docs/features/argument_processing.rst

```diff
@@ -40,11 +40,11 @@ Decorators provided by cmd2 for argument processing
 ``cmd2`` provides the following decorators for assisting with parsing arguments
 passed to commands:
 
-.. automethod:: cmd2.cmd2.with_argument_list
+.. automethod:: cmd2.decorators.with_argument_list
```

Comment:
I wonder if it should be `autofunction` instead of `automethod` here?  Or should it be `automethod` in the **api/decorators.rst** file?  Or does it just not matter?